ui/src/components/tag/TagList.js

```diff
@@ -217,6 +217,7 @@ class TagList extends React.Component {
                 tags: collectionDetails.tags,
             };
         }
+        return collectionDetails;
```

Comment:
so return collectionDetails when category = 'policy' ?

Reply:
My next pr is service with tags so it will be used for policy and service

Reply:
maybe add a comment here mentioning `return collectionDetails when category is policy or service` since all the other cases have an entity mapped with the switch case